application/common/models/User.php

```diff
@@ -414,6 +414,32 @@ public function getVerifiedMfaOptions()
         return $mfas;
     }
 
+    /*
+     * @return bool
+     */
+    public function hasMfaBackupCodes()
+    {
+        foreach ($this->getVerifiedMfaOptions() as $mfaOption) {
+            if ($mfaOption->type == Mfa::TYPE_BACKUPCODE) {
+                return true;
+            }
+        }
+        return false;
+    }
+
+    /*
+     * @return int the count of a user's Mfa backup codes
+     */
+    public function countMfaBackupCodes()
```

Comment:
We were already showing this number on the PW Manager profile page. Do we need to update code somewhere else in ID Broker to also use this function now (so that the logic stays the same for both)?

Reply:
Leaving as is, after discussing it.